app/adapters/config.js

```diff
@@ -4,10 +4,10 @@ import Ember from 'ember';
 import PouchAdapterUtils from 'hospitalrun/mixins/pouch-adapter-utils';
 
 export default Adapter.extend(PouchAdapterUtils, {
-    databaseName: 'config',    
-    db:  Ember.computed.alias('pouchDBService.configDB'),
-    pouchDBService: Ember.inject.service('pouchdb'),
-    
+    databaseName: 'config',
+    database: Ember.inject.service(),
+    db:  Ember.computed.alias('database.configDB'),
```

Comment:
Good catch. `db` is used by PouchAdapter, which I thought this adapter extends from, but it doesn't. I removed the reference to `db` in latest commit.